COMMON_ISSUES.md

```diff
@@ -8,6 +8,42 @@ known or a workaround is available. Please also take a look at the
 release. If your issue hasn't been reported, consider submitting
 [a new issue](https://github.com/twilio/twilio-video.js/issues/new).
 
+Working around the browsers' autoplay policy
```

Comment:
browsers' -> browser's

Reply:
@syerrapragada `browsers'` signifies more than one browser, so this is the correct way to use an apostrophe.